packages/model-viewer/src/features/ar.ts

```diff
@@ -198,7 +198,7 @@ configuration or device capabilities');
       console.log('Attempting to present in AR...');
 
       try {
-        await this[$renderer].present(this[$scene]);
+        await this[$renderer].arRenderer.present(this[$scene]);
```

Comment:
I would generally recommend against this kind of pattern, where you reach several objects deep to perform some action. The fact that `ARRenderer` is a member of `Renderer` implies that there is shared business logic or state between the two. However, by directly affecting the state of `ARRenderer` in this way, you are creating surface area for contention over that state. Even if it works fine today, it is ripe for bugs in the future.

I would recommend either continuing to wrap the calls into `ARRenderer`, or else factoring it out of `Renderer` entirely so that the contract between `ARRenderer` and `Renderer` can be expressed in terms of implementation accessed in common by this module and `Renderer`.

Reply:
Agreed; I filed #1228 to remind me to pull ARRenderer out of the Renderer class entirely.